src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/annotation/missingdeprecated/Example2.txt

```diff
@@ -1,32 +0,0 @@
-/*xml
-<module name="Checker">
-  <module name="TreeWalker">
-    <module name="MissingDeprecated">
-      <property name="violateExecutionOnNonTightHtml" value="true"/>
-    </module>
-  </module>
-</module>
-*/
-
-// xdoc section -- start
-@Deprecated
-public static final int MY_CONST = 13; // ok
-
-/** This javadoc is missing deprecated tag. */
-@Deprecated
-public static final int COUNTER = 10; // violation
-
-/**
- * @deprecated
- * <p></p>
- */
-@Deprecated
-public static final int NUM = 123456; // ok
-
-/**
- * @deprecated
- * <p>
- */
-@Deprecated
-public static final int CONST = 12; // violation, tight HTML rules not obeyed
```

Comment:
There was a violation here. And, the violation was // violation, tight HTML rules not obeyed. And, i was confused if it made any sense have to this violation here. I tried to get MSG_UNCLOSED_HTML_TAG but i couldn't the tests stating java.util.MissingResourceException

Reply:
@relentless-pursuit please confirm that this violation message is part of Javadoc parser and not this specific check.

Reply:
it might complained on not closed tag of `<p>`.
it should be simple violation message as all others.

Reply:
it is defect - https://github.com/checkstyle/checkstyle/issues/13745 

@relentless-pursuit , please restore `<p>` and put there message as is, even it is defect. It will be fixed in separate  PR.

Reply:
> @relentless-pursuit please confirm that this violation message is part of Javadoc parser and not this specific check.

yes i assume. the exepected error message is `'javadoc.unclosedHtml'` which is traced to Javadoc Parser. So, shall i wait for it to be fixed?

Because, when i try with modified test case as below after making necessary imports, 

```
    @Test
    public void testExample2() throws Exception {
        final String[] expected = {
            "20: " + getCheckMessage(MSG_KEY_ANNOTATION_MISSING_DEPRECATED),
            "32: " + getCheckMessage(MSG_UNCLOSED_HTML_TAG, "p"),
        };
```
I get a missing resource exception.

`java.util.MissingResourceException: Can't find resource for bundle java.util.PropertyResourceBundle, key javadoc.unclosedHtml`

Reply:
do not wait, lets use it as it is.
On moment when such issue fixed, your Example will blowup in failure and author of fix will update your example.

may be you can fix this issue after merge of this PR?

Reply:
make line:
`"32: " + MSG_UNCLOSED_HTML_TAG `

Reply:
problem is that name of message is not resolved to its content, so for now we print user name of message :), not a message.

Reply:
> do not wait, lets use it as it is. On moment when such issue fixed, your Example will blowup in failure and author of fix will update your example.
> 
> may be you can fix this issue after merge of this PR?

Sure. Will try my best.

Reply:
> make line: `"32: " + MSG_UNCLOSED_HTML_TAG `

done